packages/pakistan/README.md

```diff
@@ -0,0 +1,181 @@
+# 🇵🇰 Paksitan
+
+Contains all the provinces of Pakistan:
+* Azad Jammu Kashmir
+* FATA
+* Gilgit Baltistan
+* Balochistan
+* Khyber Pakhtunkwah
+* Punjab
+* Sindh
+
+Contains all the cities of Pakistan:
```

Comment:
They seem to be called "districts" (source: https://en.wikipedia.org/wiki/Districts_of_Pakistan)

Reply:
Districts they are. :/ Fixing that.